ReactCommon/cxxreact/Instance.cpp

```diff
@@ -48,7 +48,8 @@ void Instance::initializeBridge(
   jsQueue->runOnQueueSync([this, &jsef, jsQueue]() mutable {
     nativeToJsBridge_ = std::make_unique<NativeToJsBridge>(
         jsef.get(), moduleRegistry_, jsQueue, callback_);
-
+    // TODO investigate why it has to be async
```

Comment:
Why what has to be async? InitializeRuntime?

Reply:
I think this is the old leftover, where we make the initial drafts of changes for RN 0.60. When implementing it, we've noticed that the code in `NativeToJsBridge::initializeRuntime` has to be run using `runOnExecutorQueue`. We haven't investigated more why we have to do it like that, since the code was working as expected and was crashing if `runOnExecutorQueue` was not used. This comment can be discarded and we'll remove it.

Reply:
With some versions of JSC, if you create the runtime on one thread, and then use it on another, GC bugs will cause crashes.  That's why this has to be done this way.  It's not pretty, but there's not much else to be done if you want to use older versions of iOS.  I'm not sure if the android JSC has this bug or not.